back-end/src/main/java/com/skynet/skymed/controller/PessoaController.java

```diff
@@ -25,59 +27,76 @@
 public class PessoaController {
 
 	private EmailDePacienteService servicoDeEmailPaciente = new EmailDePacienteService();
+	private final String PACIENTE_INEXISTENTE = "Paciente inexistente.";
 
 	@Autowired
 	private PessoaRepository pessoaDB;
+	
+	@ExceptionHandler({ HttpMessageNotReadableException.class })
+    public ResponseEntity<Object> handleException(HttpMessageNotReadableException ex) {
+		return ResponseEntity.badRequest().body(ex.getMostSpecificCause().getMessage());
+    }
 
 	@GetMapping
-	public ResponseEntity<ArrayList<Pessoa>> getObject() {
+	public ResponseEntity<Object> getObject() {
 		var pessoas = pessoaDB.findAll();
-		if (pessoas.size() == 0) {
-			return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
 
+		if (pessoas.size() == 0) {
+			return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Nenhuma pessoa encontrada.");
 		}
 		return ResponseEntity.ok((ArrayList<Pessoa>) pessoas);
 	}
 
 	@PostMapping
-	public ResponseEntity<Pessoa> postPessoa(@RequestBody Pessoa object) throws Exception {
+	public ResponseEntity<Object> postPessoa(@RequestBody Pessoa object) throws Exception {
 		if (object.getId() != null) {
 			var pessoa = getById(object.getId().intValue());
 
-			if (pessoa.hasBody()) {
-				return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
-
+			if (pessoa.getBody().equals("Paciente_INEXISTENTE")) {
+				return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Paciente existente.");
 			}
+		} else if (pessoaDB.verificaCpfExistente(object.getCpf()) != null) {
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Cpf existente.");
+
+		} else if (pessoaDB.verificaRgExistente(object.getRg()) != null) {
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("RG existente.");
+
+		} else if (pessoaDB.verificaEmailExistente(object.getEmail()) != null) {
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("E-mail existente.");
+
 		}
 
-		servicoDeEmailPaciente.enviaEmail(object);
+		// servicoDeEmailPaciente.enviaEmail(object);
 		pessoaDB.save(object);
-		return ResponseEntity.ok(object);
+		return ResponseEntity.ok("Paciente cadastrado com sucesso");
```

Comment:
```suggestion
		return ResponseEntity.ok(object);
```
Vai enviar o 200 como erro, coloque o objeto ou null.